queries.sql

```diff
@@ -23,3 +23,38 @@ SELECT * from animals WHERE NOT name='Gabumon';
 
 --Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
 SELECT * from animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+BEGIN;
+UPDATE animals SET species='unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+
+SELECT * FROM animals;
+UPDATE animals SET species='pokemon' WHERE species IS NULL;
+SELECT * FROM animals;
```

Comment:
- [ ] These queries should be inside a transaction. Kindly `BEGIN;` a transaction before updating the table. Also, `SELECT species from animals` instead of `SELECT * from animals`.